### PR TITLE
WT-9821 Add option to verify to report all data corruption in a file (v5.0 backport)

### PR DESCRIPTION
**Pull request by etienneptl**
Note that this PR contains the fallout WT-10601 (Fix wt verify -c failure when first block on page is corrupt)

### CODE REVIEW

**Review by lukech — APPROVED** [review]
LGTM, thanks

File: src/utilities/util_verify.c [hunk 1]
```diff
@@ -12,11 +12,14 @@ static int
 usage(void)
 {
     static const char *options[] = {"-d config",
-      "display underlying information during verification", "-s",
-      "verify against the specified timestamp", NULL, NULL};
+      "display underlying information during verification", "-c",
+      "continue to the next page after encountering error during verification", "-s",
+      "verify against the specified timestamp", "-t", "do not clear txn ids during verification",
```
Comment (mm-ng) [inline src/utilities/util_verify.c:17]: Why are we adding the text related to `-t` option here? I don't think this functionality exists in the verify utility for version 5.0.

**Review by mm-ng — COMMENTED** [review]
This is looking good in general, I've just added one comment.

File: src/utilities/util_verify.c (hunk 1 above)
Comment (etienneptl) [reply src/utilities/util_verify.c:17]: This is a mistake, sorry. Thank you for finding this.

Comment (etienneptl) [reply src/utilities/util_verify.c]: I have removed it from the backport, should be gone now.

**Review by mariammojid — APPROVED** [review]
LGTM :)

**Review by mm-ng — APPROVED** [review]
LGTM

**Pull request merged**